app/models/container.rb

```diff
@@ -4,10 +4,7 @@ class Container < ActiveRecord::Base
 
   belongs_to :compute_resource
   belongs_to :registry, :class_name => "DockerRegistry", :foreign_key => :registry_id
-  has_many :environment_variables, :dependent  => :destroy, :foreign_key => :reference_id,
-                                   :inverse_of => :container,
-                                   :class_name => 'EnvironmentVariable',
-                                   :validate => false
+  has_many :environment_variables, :dependent  => :destroy, :foreign_key => :reference_id, :inverse_of => :container, :class_name => 'EnvironmentVariable', validate: false
```

Comment:
Line is too long. [171/100]<br>Unnecessary spacing detected.<br>Always use hash rockets in hashes.